### PR TITLE
Hadoop also handle ".tiff" extension

### PR DESCRIPTION
**Pull request by zifeo**
Hadoop filesystem now matches on one or more specified extensions.

### CODE REVIEW

**Comment by pomadchin** [comment]
There can also be tif extensions in the uppercase, mb to make smth like `toLowerCase` for the path string, or just to add upper case extensions into extensions seq?

**Comment by zifeo** [comment]
Good point, thanks! The uppercase versions make also sens.

File: spark/src/main/scala/geotrellis/spark/io/hadoop/HadoopSparkContextMethods.scala [hunk 1]
```diff
@@ -11,18 +11,27 @@ import org.apache.hadoop.mapreduce.Job
...
+
+  def hadoopGeoTiffRDD(path: Path, tiffExtension: String): RDD[(ProjectedExtent, Tile)] =
+    hadoopGeoTiffRDD(path, Seq(tiffExtension))
+
+  def hadoopGeoTiffRDD(path: Path, tiffExtensions: Seq[String]): RDD[(ProjectedExtent, Tile)] = {
     val searchPath = path.toString match {
-      case p if p.contains(tiffExtension) => path
-      case p => new Path(s"$p/*$tiffExtension")
+      case p if tiffExtensions.exists(p.endsWith) => path
+      case p => new Path(p + "/*" + tiffExtensions.mkString("{", ",", "}"))
```
Comment (lossyrob) [inline spark/src/main/scala/geotrellis/spark/io/hadoop/HadoopSparkContextMethods.scala]: case p =>
  val extensions =  tiffExtensions.mkString("{", ",", "}")
  new Path(s"$p/*$extensions")

**Comment by lossyrob** [comment]
+1

**Pull request merged**